lib/MusicBrainz/Server/Controller/Edit.pm

```diff
@@ -102,7 +102,7 @@ sub data : Chained('load') RequireAuth
         component_path => 'edit/EditData',
         component_props => {
             edit => $edit->TO_JSON,
-            rawData => $c->json->pretty->encode($decoded_raw_data),
+            rawData => $c->json->canonical->pretty->encode($decoded_raw_data),
```

Comment:
Oops, I'm pretty sure doing `->canonical->pretty->` permanently sets that on `$c->json` for the rest of the request.  Which might be fine if this is the only use during the request, but it's not meant to have settings changed on it. (Note: suggestion needs `use JSON;`.)

```suggestion
            rawData => JSON->new->canonical->pretty->encode($decoded_raw_data),
```

Reply:
`$c->json` also has `allow_blessed` and `convert_blessed` enabled, but I don't think those are needed here.

Reply:
Oh! Ok, updated :)